_commands/tools/units.md

```diff
@@ -0,0 +1,77 @@
+---
+---
+
+units
+--
+
+`units` gives conversion rates from one scale to another. 
+
+~~~ bash
+$ units [-f unitsfile] [-q] [-v] [from-unit to-unit]...
+~~~
+
+<!--more-->
+
+The units are defined in an external data file. You can use the extensive data file that comes with this program, or you can provide your own data file to suit your needs. You can use the program interactively with prompts, or you can use it from the command line.
+
+will list the value you will need to multiply (*) or divide (/) your original number by to convert to your desired units listed.
```

Comment:
Updated - thanks for the catch, I didn't realized I had changed the ordering without making sure the full sentences still made sense.